site/en/setup.md

```diff
@@ -94,6 +94,29 @@ keep the software versions up-to-date and ensure that all components are at a us
     
 ## ocrd_all via Docker
 
+### Prerequisites
+
+If you want to use the OCR-D-Docker-solution, [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-using-the-repository) and [docker compose](https://docs.docker.com/compose/install/) have to be installed.
+
+After installing docker you have to set up daemon and add user to  the group 'docker'
```

Comment:
```suggestion
After installing docker you have to set up the docker daemon and add your user to the `docker` group:
```

Reply:
...may have to set up and start... (depending on installation method, this might already have been done at this point)